### PR TITLE
Priority picture request revamp

### PR DESCRIPTION
**Pull request by albarv340**
Renames all identifiers associated with priority picture requests (priority image, view...) to something with priority picture. 
Adds a camera icon where a picture has been requested on the map.
Adds a crosshair where the requested image will be requested.
Makes it possible to choose urgency when taking a picture.
Adds an overlay on the screen when the picture is being taken, to clarify. 
Changes the way information about a requested picture is presented in both the side menu and popup menu.

### CODE REVIEW

File: src/front-end/react_base/src/JS/Startup/StartUp.js [hunk 1]
```diff
@@ -147,9 +147,14 @@ function StartUp(props) {
    */
   function calculateBounds() {
     let bounds = Leaflet.latLngBounds(props.store.areaWaypoints);
-
-    let topLeft = [bounds.getNorth(), bounds.getWest()];
-    let bottomRight = [bounds.getSouth(), bounds.getEast()];
+    const paddingX = 3 * (bounds.getEast() - bounds.getWest());
```
Comment (Edvin-Dyremark) [inline src/front-end/react_base/src/JS/Startup/StartUp.js:150]: Add comment about padding here

File: src/front-end/react_base/src/JS/Storage.js [hunk 2]
```diff
@@ -54,10 +54,7 @@ const startView = {
...
     }
  */
-var initialRequestQueue = {
-  size: 0,
-  items: [],
-};
+
 /* Elements are structured like following example:
 {
     "type" : #Choice "RGB/IR",
```
Comment (Edvin-Dyremark) [inline src/front-end/react_base/src/JS/Storage.js:60]: Type not used

File: src/front-end/react_base/src/JS/Menu/PriorityPicturesTab.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        </ColorWrapper>
+      </ListItem>
+
+      <Divider />
+
+      <List>
+        {/* Sort requests on request time, then add all to list */}
+        {[...props.store.pictureRequestQueue]
+          .sort((i1, i2) => i2.requestTime - i1.requestTime)
+          .map((item, index) => (
```
Comment (Edvin-Dyremark) [inline src/front-end/react_base/src/JS/Menu/PriorityPicturesTab.js:58]: This causes an error: each child should have a unique key in a list. 
Give each item a unique key.

File: src/front-end/react_base/src/JS/Startup/StartUp.js [hunk 4]
```diff
@@ -147,9 +147,14 @@ function StartUp(props) {
...
+    const paddingY = 3 * (bounds.getSouth() - bounds.getNorth());
```
Comment (Edvin-Dyremark) [inline src/front-end/react_base/src/JS/Startup/StartUp.js:151]: Map zooms out too much when defining area.

**Review by Edvin-Dyremark — CHANGES_REQUESTED** [review]

File: src/front-end/react_base/src/JS/Connection/Upstream.js [hunk 5]
```diff
@@ -3,7 +3,7 @@
  */
 
 import ServerConnection from "./ServerConnection.js";
-import { store, receiveRequest } from "../Storage.js";
+import { store, receivePictureRequest } from "../Storage.js";
 
 /**
```
[the remark this answers is not in the document]
Comment (albarv340) [reply src/front-end/react_base/src/JS/Connection/Upstream.js:8]: This has to do with the regular images, so it might still be needed

**Pull request merged**